src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -63,6 +65,21 @@ export default App.extend({
       Radio.request('entities', 'fetch:files:collection:byAction', this.action.id),
     ];
   },
+  onWebsocketMessage({ category, payload }) {
+    if (category !== 'CommentAdded') return;
+
+    const { comment, relationships, attributes } = payload;
```

Comment:
clinician wouldn't be included here it'd be the author https://github.com/RoundingWell/care-ops-backend/blob/2c937bc0180ffe7aefe6ba15449ad5d246cd1f6d/tests/Notifications/Data/Messages/comment-added-message-structure.json#L4

Reply:
we can set `created_at` to `dayjs().now()` and leave `edited_at` null.. this'd be fine, but until we can handle comment editing, we probably should handle adding, and that to me is much more complicated